Formula/wellington.rb

```diff
@@ -17,72 +15,41 @@ class Wellington < Formula
     sha256 "2ef26afb326f22102249daee8c795fb467ce9716a2aae306efd23efc65020df3" => :mavericks
   end
 
-  devel do
-    url "https://github.com/wellington/wellington/archive/v1.0.0-beta1.tar.gz"
-    sha256 "6ea2a260ba7146a6bd87f42ab22082dfd84eb5aa52adae0629cbe71395cf56de"
-    version "1.0.0-beta1"
-  end
-
   needs :cxx11
 
-  head do
-    url "https://github.com/wellington/wellington.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   depends_on "go" => :build
-  depends_on "godep" => :build
   depends_on "pkg-config" => :build
 
-  go_resource "github.com/kr/fs" do
-    url "https://github.com/kr/fs.git",
-        :revision => "2788f0dbd16903de03cb8186e5c7d97b69ad387b"
-  end
-
-  go_resource "golang.org/x/tools" do
-    url "https://github.com/golang/tools.git",
-        :revision => "ea5101579e09ace53571c8a5bae6ebb896f8d5e4"
+  go_resource "golang.org/x/net" do
+    url "https://github.com/golang/net.git",
+        revision: "f09c4662a0bd6bd8943ac7b4931e185df9471da4"
   end
 
   def install
     ENV.cxx11 if MacOS.version < :mavericks
 
-    version = File.read("version.txt").chomp
     ENV["GOPATH"] = buildpath
-    Language::Go.stage_deps resources, buildpath/"src"
-
-    system "godep", "restore"
+    version = File.read("version.txt").chomp
```

Comment:
You should be able to get this by using `version`, which is already defined.
If we're in the stable spec:
```
irb(wellington):001:0> version
=> #<Version::FromURL:0x007fb28a37eb98 @version="1.0.3">
```
If we're in the head spec
```
irb(wellington):001:0> version
=> #<HeadVersion:0x007fc7ce0dcd78 @version="HEAD-2361488", @commit="2361488">
irb(wellington):002:0> stable.version
=> #<Version::FromURL:0x007fc7ce168648 @version="1.0.3">
irb(wellington):003:0> 
```